packages/web-components/src/player-component/player.class.ts

```diff
@@ -332,12 +331,12 @@ export class PlayerWrapper {
 
         // Remove timeline
         this.removeTimelineComponent();
-        // await this.controls.destroy();
 
         if (this.isLoaded) {
             this.video.pause();
             this.video.src = '';
             await this.player?.unload();
+            await this.controls.destroy();
         }
         this.isLoaded = false;
```

Comment:
You can tuck this into the if-statement block above since it is only needed if this.isLoaded was true, otherwise it is already false.